index.html

```diff
@@ -3,11 +3,44 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="/style.css"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+        <div id="modal-popup">
+            <table id="modal-popup-table">
+                <tr><td colspan="2"><div id="modal-popup-text">Hello world!</div></td></tr>
+                <tr>
+                    <td>
+                        <button id="modal-popup-cancel">Cancel</button>
+                    </td>
+                    <td id="cell-modal-popup-ok">
+                        <button id="modal-popup-ok">OK</button>
+                    </td>
+                </tr>
+            </table>
+        </div>
+        <div id="tooltip-on-hover">
```

Comment:
Too many `ids`.
It is ok for this homework, but if you use id for elements to set behaviour - your code will hard to change or extend.
The good way is to use class and [data-attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) to define behaviour for elements. And then, you can change your HTML, place elements to any place and it will work the same

Don't afraid, most part of student did the same thing 😉